README.md

```diff
@@ -17,440 +14,20 @@ Platform support:
 
 ![Uno Material](./images/UnoMaterial.png)
 
-Uno Material Design Guideline is a resource for designers and software developers that combines Material and Uno design guidance in single document. It is an easy way to kickstart design and implementation of cross-platform experiences with unified Material design system look and feel, using Sketch and Uno Platform.
-
-Download the [Uno Platform Design Guidelines sketch file](./design/Uno-Platform-Design-Guidelines.sketch) to get started.
-
-It includes:
-- Uno-Material components
-- Uno type resource names
-- Uno asset naming and export guidance
+<!-- TODO : Insert build status -->
```

Comment:
I don't we need the extra lines after the ToDo ?

Reply:
Which empty line? L17is the todo and L18 is the start of table already